src/test/java/POM/AllSongsPage.java

```diff
@@ -45,5 +45,11 @@ public WebElement hoverPlay() {
         actions.moveToElement(play).perform();
         return play;
     }
+
+    public AllSongsPage clickOnAllSongs(){
+        driver.findElement(allSongsMenuItemLocator).click();
+        return new AllSongsPage(driver);
+    }
+
```

Comment:
1. private final By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
  private final By firstSongLocator = By.cssSelector(".play");
  
  2. Add possibility of creating Locators with variables, for example
  
 ` private  By soundBarPlay = By.cssSelector("[data-testid = 'sound-bar-play']");`
 
```
   public boolean isSongPlaying() {
    WebElement soundBarVisualizer = driver.findElement(soundBarPlay);
    return soundBarVisualizer.isDisplayed();
  }
```